test/e2e/storage/volume_provisioning.go

```diff
@@ -716,6 +786,37 @@ var _ = utils.SIGDescribe("Dynamic Provisioning", func() {
 			Expect(claim.Status.Phase).To(Equal(v1.ClaimPending))
 		})
 	})
+
+	framework.KubeDescribe("GlusterDynamicProvisioner", func() {
+		It("should create and delete persistent volumes [fast]", func() {
+			By("creating a Gluster DP server Pod")
+			pod := startGlusterDpServerPod(c, ns)
+			serverUrl := "https://" + pod.Status.PodIP + ":8081"
+
+			By("creating a StorageClass")
+			test := storageClassTest{
+				name:         "Gluster Dynamic provisioner test",
+				provisioner:  "kubernetes.io/glusterfs",
+				claimSize:    "1500Mi",
+				expectedSize: "1500Mi",
+				parameters:   map[string]string{"resturl": serverUrl},
+			}
+			class := newStorageClass(test, ns, "gluster")
+			_, err := c.Storage().StorageClasses().Create(class)
```

Comment:
The sc is created here and again in `testDynamicProvisioning`

Reply:
Yeah, SC and Claim creation in testglusterdynamicprovisioning has been disabled now. I will check the test result again and wipe the commented code accordingly.